services/tilesets.js

```diff
@@ -16,6 +16,12 @@ var Tilesets = {};
  *
  * @param {Object} [config]
  * @param {string} [config.ownerId]
+ * @param {Object} [query]
+ * @param {string} [query.type] - Filter results by tileset type, either `raster` or `vector`.
+ * @param {number} [query.limit] - The maximum number of tilesets to return, from 1 to 500. The default is 100.
```

Comment:
you can specify the default within the jsdoc like https://github.com/mapbox/mapbox-sdk-js/blob/master/services/geocoding.js#L44